library/src/main/java/com/okta/appauth/android/OktaManagementActivity.java

```diff
@@ -104,40 +99,47 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     protected void onStart() {
         super.onStart();
-
-        if (mExecutor.isShutdown()) {
-            mExecutor = Executors.newSingleThreadExecutor();
-        }
-
         OAuthClientConfiguration config = OAuthClientConfiguration.getInstance(this);
         if (config.hasConfigurationChanged()) {
             signOut();
             return;
         }
+        // the stored AuthState is incomplete, so check if we are currently receiving the result of
+        // the authorization flow from the browser.
+        AuthorizationManagementResponse response =
+                AuthorizationManagementResponse.fromIntent(getIntent());
+        AuthorizationException ex = AuthorizationException.fromIntent(getIntent());
+
+        if (ex != null || response == null) {
+            Log.i(TAG, "Authorization flow failed: " + ex);
```

Comment:
info? should this just be a debug? possibly a warn? depending on if this is an indication of a invalid user credentials, or a communication error

Reply:
Changed to warning